source/includes/changelogs/releases/100.7.2.rst

```diff
@@ -0,0 +1,13 @@
+.. _100.7.2-changelog:
+
+100.7.2 Changelog
+-----------------
+
+*Released 2023-05-30*
+
+This release fixes an issue with installing Database Tools on RHEL aarch64 architecture. 
```

Comment:
I think this should be "AArch64". And, "the RHEL AArch64 ..."

Reply:
Fixed.